test/Customer-test.js

```diff
@@ -32,4 +34,23 @@ describe('Customer', () => {
     it('should have a name that is a string', () => {
       expect(customer.name).to.be.a('string');
     });
+
+    it('should have a property that holds the user bookings', () => {
+      expect(customer.bookings).to.deep.equal([])
+    });
+
+    it('should have a total amount spent by the customer on rooms', () => {
+      expect(customer.totalSpent).to.equal(0);
+    });
```

Comment:
This one seem redundant to the last test.

Reply:
L: I was debating that one as well. There was that happy path/sad path conversation in the slack channel, and Phil had mentioned that he often will test if there's nothing passed in that he'll...even as I'm typing I'm seeing how that still doesn't really fit the bill. Off it should go!

D: Could you explain what a minute test is please? I think your testing habits look good.
L: Thanks!!!! I'd heard (all the rumors, right?) that a group had been told that their tests were 'too minute' ie that they had too many. I didn't know if such a thing was real, but I know we're not about redundant code. Thought I'd check in with you and see what you thought. I will leave them until I'm told by the powers that be otherwise. 

D: I don't know that you need to check if the id is a positive number since that is supposedly computer-generated, but there could be a server side error of that variety. So maybe that is smart.
L: Duh!!! Completely forgot about that aspect of the id. But you also make a good point. Powers that be...? 

D: I checked on your booking tests as well since you called Booking in customer test. I noticed you have a test for the formatting but don't test the format, just the length of the date.
L: That would be correct- my complete inability to label anything well. I'm also using the input type that is the 'preset' date, and, as far as I understand, it automatically comes back with that format. But kind of like you said above, perhaps there could be an error at some point so best to check. I'll for sure rename that test so it's not so misleading.